parcels/particleset.py

```diff
@@ -64,6 +64,9 @@ def convert_to_list(var):
             raise NotImplementedError('If fieldset.time_origin is not a date, time of a particle must be a double')
         time = [self.time_origin.reltime(t) if isinstance(t, np.datetime64) else t for t in time]
 
+        for kwvar in kwargs:
+            kwargs[kwvar] = convert_to_list(kwargs[kwvar])
```

Comment:
Should we use a copy of kwargs or use it directly?

Reply:
Why do you think we need a copy? This won't change, would it?

Reply:
I'm always double checking when editing a dictionary, but here you're right it would not change anything.